### PR TITLE
Set user-agent from a singleton util.

### PR DESCRIPTION
**Pull request by zhongyi-tong**
#172

### CODE REVIEW

File: app/renderer/js/utils/system-util.js [hunk 1]
```diff
@@ -32,8 +43,12 @@ class SystemUtil {
 		}
 	}
 
+	setUserAgent(webViewUserAgent) {
+		this.userAgent = 'ZulipElectron/' + webViewUserAgent;
```
Comment (akashnimare) [inline app/renderer/js/utils/system-util.js]: Prettify -
```javascript
		this.userAgent = 'ZulipElectron' + ' ' + webViewUserAgent;

**Review by akashnimare — CHANGES_REQUESTED** [review]
Also - https://travis-ci.org/zulip/zulip-electron/jobs/241772249#L293

**Comment by zhongyi-tong** [comment]
`"ZulipElectron/0.5.10 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Zulip/0.5.10 Chrome/56.0.2924.87 Electron/1.6.8 Safari/537.36"`

@akashnimare Added the mission app version. Cross-check please 😄

**Comment by akashnimare** [comment]
LGTM. Merged. Thanks, @geeeeeeeeek.

**Pull request merged**